README.md

```diff
@@ -9,100 +9,511 @@ support@appsflyer.com
 
 Add to your build.gradle file:
 
 ```
-implementation 'com.appsflyer:purchase-connector:0.1.0'
+implementation 'com.appsflyer:purchase-connector:0.2.0'
 implementation 'com.android.billingclient:billing:$play_billing_version'
 ```
 
-where play_billing_version is 3.x.x. Note, version 4 of Billing Client is not supported!
+where play_billing_version is 3.x.x. Note, versions 4 and 5 of Billing Client are not supported!
+
+## <a id="basic-integration"> Basic Integration Of The SDK
+### <a id="create-instance"> Create PurchaseClient Instance 
+Create an instance of this SDK to configure (in the following steps) for observing and validating transactions in your app. </br>
+**Make sure to save a reference to the built object. If the object is not saved, it could lead to unexpected behavior and memory leaks.** </br>
 
-### 🚀 Basic integration of the SDK
-#### Java
+* Kotlin
+```kotlin
+        // init
+        val builder = PurchaseClient.Builder(this, Store.GOOGLE)
+        // Make sure to keep this instance
+        val afPurchaseClient = builder.build()
 ```
-new PurchaseClient.Builder(
-                // context
-                this,
-                // Store enum
-                Store.GOOGLE)
-                // Enable subs auto logging
-                .logSubscriptions(true)
-                // Set sandbox (false by default)
-                .setSandbox(true)
-                // Purchase Event Data source listener. Invoked before sending data to AF servers
-                // to let customer add extra parameters to the payload
-                .setPurchaseEventDataSource(list -> {
-                    Map<String, Object> additionalParams = new HashMap<>();
-                    Log.d(LOG_TAG, "[PurchaseConnector]: Data Source invoked");
-                    for (PurchaseEvent p : list) {
-                        Log.d(LOG_TAG, "SKU = " + p.getSku());
-                        additionalParams.put("customParam", p.getSku());
+
+* Java
+```java
+        // init
+        PurchaseClient.Builder builder = new PurchaseClient.Builder(context, Store.GOOGLE);
+        // Make sure to keep this instance
+        PurchaseClient purchaseClient = builder.build();
+```
+
+### <a id="start"> Start Observing Transactions
+Start the SDK instance to observe transactions. </br>
+* Kotlin
+```kotlin
+        // start
+        afPurchaseClient.startObservingTransactions()
+```
+
+* Java
+```java
+        // start
+        afPurchaseClient.startObservingTransactions();
+```
+
+### <a id="stop"> Stop Observing Transactions
+Stop the SDK instance from observing transactions. </br>
+* Kotlin
+```kotlin
+        // start
+        afPurchaseClient.stopObservingTransactions()
+```
+
+* Java
+```java
+        // start
+        afPurchaseClient.stopObservingTransactions();
+```
+
+### <a id="log-subscriptions"> Log Subscriptions
+Enables automatic logging of subscription events. </br>
+Pass true to enable, false to disable. Disabled by default</br>
```

Comment:
"Disabled by default" means that if the API is not used, the feature will be disabled, correct? I am not sure it is clear enough, and we should clarify that.